dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -314,12 +314,12 @@ if [ "$RH_CHE_USE_CUSTOM_REGISTRIES" == "true" ]; then
     oc delete configmap plugin-registry > /dev/null 2>&1
   fi
   echo "Downloading che plugin registry yaml"
-  if ! (curl -L0fs https://raw.githubusercontent.com/eclipse/che-plugin-registry/master/deploy/openshift/che-plugin-registry.yml -o che-plugin-registry.yaml > /dev/null 2>&1); then
+  if ! (curl -L0fs https://raw.githubusercontent.com/eclipse/che-plugin-registry/7.26.0/deploy/openshift/che-plugin-registry.yml -o che-plugin-registry.yaml > /dev/null 2>&1); then
```

Comment:
replace `7.26.0` with `${RH_CHE_VERSION}`

Reply:
it will not work because there is no snapshot version in the repository